vector/src/main/res/layout/fragment_session_overview.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I would have used a ScrollView here.

Reply:
Yes it might be a good idea for small devices. It will be done in a next PR (session details screen).